### ISSUE TITLE
Validating linked URLs

### ISSUE THREAD

**Issue by marcoscaceres** [tobie/specref#678]
I wonder if we should have a periodic (e.g., one per week/month/year?) check of the references to see which are 404?

**Comment by tobie** [comment]
Yeah. What do we do with 404, though? Seems we'd need to think about what that implies first.

**Comment by marcoscaceres** [comment]
Yeah, lots of options... just thinking out loud here: 

 1. just make a "404s.txt" somewhere, but do nothing with the actual biblio data.
 1. file bugs here and ping the contributor identified via `git blame`.
 1. Copy the `href` value to a new prop name `old_href` or whatever, so tools no longer link. 
 1. For each 404, see if it's available still on Web Archive and use that instead.
 1. Do nothing... it's the web... stuff breaks and that's ok.
